igvm/hypervisor.py

```diff
@@ -817,3 +832,57 @@ def device_to_netcat(self, device, size, listener):
             '| /bin/nc.openbsd -q 1 {2} {3}'
             .format(device, size, *listener)
         )
+
+    def hv_pcpus_used_by_vm(self, vm) -> float:
```

Comment:
If we support type hints already you can also use some for the parameters (You may want to check the following methods as well).

Reply:
Added missing type hints.